packages/auth/src/grant/routes.ts

```diff
@@ -198,10 +277,14 @@ async function continueGrant(
   const continueToken = (ctx.headers['authorization'] as string)?.split(
     'GNAP '
   )[1]
-  const { interact_ref: interactRef } = ctx.request.body
 
-  if (!continueId || !continueToken || !interactRef) {
-    ctx.throw(401, { error: 'invalid_request' })
+  if (!continueId || !continueToken) {
+    ctx.throw(401, {
+      error: {
+        code: 'invalid_continuation',
```

Comment:
maybe we can provide these error codes as the second argument to `ctx.throw` such that we get them served up to us in the response: https://github.com/koajs/koa/blob/master/docs/api/context.md#ctxthrowstatus-msg-properties